MapboxNavigation/NavigationMapView.swift

```diff
@@ -135,6 +248,49 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         resumeNotifications()
     }
     
+    deinit {
+        suspendNotifications()
+    }
+    
+    //MARK: - Overrides
+    
+    open override func prepareForInterfaceBuilder() {
+        super.prepareForInterfaceBuilder()
+        
+        let image = UIImage(named: "feedback-map-error", in: .mapboxNavigation, compatibleWith: nil)
+        let imageView = UIImageView(image: image)
```

Comment:
We should safely unwrap the `image` variable, in case it mistakenly gets updated or deleted from the project.

Reply:
I know 👍 'd earlier, but I'm going to digress on this one, for the same reason we use force-unwraps [here.](https://github.com/mapbox/mapbox-navigation-ios/pull/1063#discussion_r170130885)

The fact that it's just a `prepareForInterfaceBuilder` call got me thinking for a minute, but `feedback-map-error` is used at runtime as well, so it seems okay that we'd assert that it exists here.